heed/src/env.rs

```diff
@@ -797,7 +811,7 @@ mod tests {
         eprintln!("env closed successfully");
 
         // Make sure we don't have a reference to the env
-        assert!(env_closing_event(&dir.path()).is_none());
+        assert!(env_closing_event(&dir.path()).unwrap().is_none());
```

Comment:
Did the `unwrap()` in tests since the api of `env_closing_env` changed to `Result<Option<EnvClosingEvent>>` Shall i avoid `unwrap` in tests?

No problem if it's the policy! :)

Reply:
Ho! That's the tests sorry! I was in the subway, hard to review! All good 😄